libs/datasets/timeseries.py

```diff
@@ -1001,6 +1006,57 @@ def drop_new_case_outliers(
     return new_timeseries
 
 
+def drop_tail_positivity_outliers(
+    dataset: MultiRegionDataset,
+    zscore_threshold: float = 10.0,
+    diff_threshold_ratio: float = 0.015,
+) -> MultiRegionDataset:
+    """Identifies and drops outliers from the new case series.
+
+    Args:
+        dataset: Timeseries.
```

Comment:
This line doesn't help the reader and seems unnecessary. What's the downside of removing it?